nuclia/lib/utils.py

```diff
@@ -0,0 +1,16 @@
+from typing import Union
+
+import httpx
+import requests
+
+from nuclia.exceptions import UserTokenExpired
+
+
+def handle_http_errors(response: Union[httpx.Response, requests.models.Response]):
+    if (
+        response.status_code == 403
+        and "Hydra token is either unexistent or revoked" in response.text
+    ):
+        raise UserTokenExpired()
+    else:
```

Comment:
should this be `elif response.status_code > 399:`?

Reply:
or just call `handle_http_errors` `if response.status_code < 399`

Reply:
oh shit, right!